setup.py

```diff
@@ -40,33 +40,40 @@ def find_name():
     raise RuntimeError('Unable to find name string.')
 
 
-setup(name=find_name(),
-      version=find_version(),
-      description='Turn complex GraphQL queries into optimized database queries.',
-      url='https://github.com/kensho-technologies/graphql-compiler',
-      author='Kensho Technologies, LLC.',
-      author_email='graphql-compiler-maintainer@kensho.com',
-      license='Apache 2.0',
-      packages=find_packages(exclude=['tests*']),
-      install_requires=[
-          'arrow>=0.7.0',
-          'funcy>=1.6',
-          'graphql-core==1.1',
-          'pytz>=2016.10',
-          'six>=1.10.0',
-      ],
-      classifiers=[
-          'Development Status :: 5 - Production/Stable',
-          'Topic :: Database :: Front-Ends',
-          'Topic :: Software Development :: Compilers',
-          'Intended Audience :: Developers',
-          'License :: OSI Approved :: Apache Software License',
-          'Programming Language :: Python :: 2',
-          'Programming Language :: Python :: 2.7',
-          'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.5',
-          'Programming Language :: Python :: 3.6',
-      ],
-      keywords='graphql database compiler orientdb',
-      python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*',
-      )
+def find_long_description():
+    """Return the content of the README.md file."""
+    return read_file('../README.md')
```

Comment:
It looks like we run the `make_new_release.sh` from the top-level of the repo, so wouldn't this be `./README.md`?

Reply:
I don't think so -- `read_file()` attempts to read a "package" file from inside the `graphql_compiler` directory of the repo ([line 19](https://github.com/kensho-technologies/graphql-compiler/pull/133/files#diff-2eeaed663bd0d25b7e608891384b7298R19)), so we have to go up a directory from there. That's why `find_name` and `find_version` can read `__init__.py` directly.

Reply:
Oh yes you're right, I missed that. Looks good then.